console/console-init/ui/mock-console-server/schema.js

```diff
@@ -253,6 +279,15 @@ const typeDefs = gql`
         "Returns the address plans defined by the system optionally filtering those for a matching address space plan and/or address type"
         addressPlans(addressSpacePlan: String, addressType: AddressType): [AddressPlan_admin_enmasse_io_v1beta2!]!
 
+        "Returns the authenticationServices"
+        authenticationServices: [AuthenticationService_admin_enmasse_io_v1beta1!]!
+        "Returns the addressSpaceSchema"
+        addressSpaceSchema: [AddressSpaceSchema_enmasse_io_v1beta1!]!
+        "Returns the addressSpaceSchema optionally filtering those for a matching address space plan and/or address type"
```

Comment:
The description is incorrect - plan is not an argument

Reply:
I have just updated the description.